package.json

```diff
@@ -31,25 +31,25 @@
     "test": "yarn lint"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.0.0-rc.3",
+    "@nuxt/kit": "3.0.0-rc.8",
```

Comment:
Do not pin the version of nuxt/kit please. This is on purpose on starter to use caret to be make module forward compatible.